src/dependabot/output.ts

```diff
@@ -14,15 +14,22 @@ const UPDATE_TYPES_PRIORITY = [
 ]
 
 export function set (updatedDependencies: Array<updatedDependency>): void {
-  core.info(`Outputting metadata for ${Pluralize('updated dependency', updatedDependencies.length, true)}`)
+  const dependencyNames = updatedDependencies.map(dependency => {
+    return dependency.dependencyName
+  }).join(', ');
```

Comment:
The linter is using the community style guide, I wonder if we should change this rule as not having semi-colons throws me a bit too.

Might be better using [the Google styleguide](https://google.github.io/styleguide/tsguide.html)

Reply:
Ah yeah, we could also use `prettier` which auto-formats with semicolons and use a linter to match those rules. It's what I've ended up using in the past.

Reply:
Ah things have changed a bit, looks like you can run `prettier --check .` in ci https://prettier.io/docs/en/integrating-with-linters.html

Reply:
I can PR a change for that if you like?